tox.ini

```
@@ -11,7 +11,7 @@ envlist = fmt, lint, unit
 src_path = {toxinidir}/src/
 tst_path = {toxinidir}/tests/
 lib_path = {toxinidir}/lib/charms/mongodb
-all_path = {[vars]src_path} {[vars]tst_path}
+all_path = {[vars]src_path} {[vars]tst_path} {[vars]lib_path}
```

Why do we play this games? The following pass lint without any issues:

```suggestion
all_path = {toxinidir}
```

---

we don't want to run checks against fetched 3rd-party libraries